### PR TITLE
Add documentation around features

### PR DESCRIPTION
**Pull request by jessicard**
Add documentation around how to create a scrapbook, how to post, and the slash commands available. Update both README and the `/about` page.

### CODE REVIEW

**Comment by lachlanjc** [comment]
#### 📝 Changed routes:


| [**/about**](https://scrapbook-56pin2kc7.hackclub.dev/about) |
|:-:|
|<a href="https://scrapbook-56pin2kc7.hackclub.dev/about">Screenshot of /about</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/1dU4fOX3eszKzxXBiLpJ7WzL6Ej1bGmzh0gqXh1C5vBpxIYll2R87uLfYS2O_vRthx8gpSWr7vBzG92Vf5OIkw.png">(view full size)</a> |







Commit 6a662eb11c464cb4377554fe1f33ed8a15415211 (https://scrapbook-56pin2kc7.hackclub.dev).

File: README.md [hunk 1]
```diff
@@ -4,15 +4,40 @@
...
 
-Behind the scenes, the site runs on [Next.js](https://nextjs.org), React.js, & [SWR](https://swr.now.sh) for data fetching. All pages are [static-rendered](https://nextjs.org/docs/basic-features/data-fetching#getstaticprops-static-generation), hosted on [Vercel](https://vercel.com). Videos are hosted by [Mux](https://mux.com). The custom domains use a [Vercel serverless function](https://github.com/hackclub/summer-domains). The [Slack integration](https://github.com/hackclub/scrappy) runs on [Express.js](https://expressjs.com), hosted on [Heroku](https://heroku.com). All the data is stored in a [PostgreSQL](https://www.postgresql.org) database, fetched using [Prisma](https://prisma.io). We built it in a week whilst preparing for the [2020 Summer of Making](https://summer.hackclub.com/).
+```
+learned Express generator today - now back to a ton of (organized) files
+```
+![Example Scrapbook post image](https://cloud-f8by3gcpv-hack-club-bot.vercel.app/0screenshot_2022-02-27_223754.png)
+
+_Example pulled from [audreyolafz's scrapbook](https://scrapbook.hackclub.com/audreyolafz)_
```
Comment (jessicard) [inline README.md:24]: @audreyolaf here's your example!

File: pages/about.mdx [hunk 2]
```diff
@@ -18,13 +18,38 @@ Originally built for the [Summer of Making](https://summer.hackclub.com/), use S
...
 
-## How does it work?
+A post requires just two things: a description and an image. Your image can be dragged straight into Slack and Scrapbook will take care of the rest. [An example message](https://hackclub.slack.com/archives/C01504DCLVD/p1646030307450419) into Slack might look like:
+
+```
+learned Express generator today - now back to a ton of (organized) files
+```
+<img src="https://cloud-f8by3gcpv-hack-club-bot.vercel.app/0screenshot_2022-02-27_223754.png" />
+
+_Example pulled from [audreyolafz's scrapbook](https://scrapbook.hackclub.com/audreyolafz)_
```
Comment (jessicard) [inline pages/about.mdx:36]: @audreyolaf also here :)

File: README.md [hunk 3]
```diff
@@ -62,10 +87,6 @@ Some “relative” colors use these colors for various components:
 
 Our dark mode is powered by [`prefers-color-scheme: dark`](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme).
 
-## Custom domains
```
Comment (jessicard) [inline README.md:65]: I removed this block since it's not available anymore. Does that seem reasonable?

File: pages/about.mdx [hunk 4]
```diff
@@ -79,17 +104,6 @@ Some “relative” colors use these colors for various components:
 
 Our dark mode is powered by [`prefers-color-scheme: dark`](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme).
 
-## Custom domains
```
Comment (jessicard) [inline pages/about.mdx:82]: Removed the custom domains block here too.

File: pages/about.mdx [hunk 5]
```diff
@@ -214,4 +228,8 @@ This site exposes a public JSON API powered by [Next.js API routes](https://next
     padding: 0 0.25em;
     border-radius: 0.125em;
   }
+
+  pre code {
```
Comment (jessicard) [inline pages/about.mdx:232]: Small fix. The padding is for inline code, but added funny padding on code blocks.

File: pages/about.mdx (hunk 4 above)
Comment (sampoder) [reply pages/about.mdx:82]: ooo this is actually now back to being available! `_Sadly, we have ran out of space and can't register any more domains._` should probably be deleted

File: pages/about.mdx (hunk 5 above)
Comment (sampoder) [reply pages/about.mdx:232]: Nice

File: pages/about.mdx (hunk 4 above)
Comment (jessicard) [reply pages/about.mdx:82]: EXCITING, i'll add it back in without that note!

**Review by sampoder — APPROVED** [review]
I love this @jessicard!

File: pages/about.mdx (hunk 4 above)
Comment (sampoder) [reply pages/about.mdx:82]: YAY!

Comment (jessicard) [reply pages/about.mdx:82]: @sampoder hows this? I assume the Vercel limitation still stands [69568c3](https://github.com/hackclub/scrapbook/pull/282/commits/69568c3a74ecfe311730fade9326f916094e08f1)

Comment (sampoder) [reply pages/about.mdx:82]: Vercel's limitation is also gone now as they took away their limits 🚀

**Comment by jessicard** [comment]
@sampoder Yes pls! 🙇

**Pull request merged**